src/genie/libs/parser/iosxe/tests/ShowClnsTraffic/cli/equal/golden_output_2_expected.py

```diff
@@ -59,9 +59,9 @@
                 "lsp_retransmissions": 0,
                 "csnp": {"level-1": {"sent": 308588, "rcvd": 149877}},
                 "psnp": {"level-1": {"sent": 0, "rcvd": 0}},
-                "dr_election": {"level-1": 9},
-                "spf_calculation": {"level-1": 1565},
-                "partial_route_calculation": {"level-1": 4},
+                "dr_election": {"level-1": 9, "level-2":7},
+                "spf_calculation": {"level-1": 1565, "level-2": 1324},
+                "partial_route_calculation": {"level-1": 4, "level-1": 6},
```

Comment:
`'level-2':6`*?

Reply:
Sure, typo. Thanks. It has been corrected